src/earn/EarnConfirmationScreen.test.tsx

```diff
@@ -138,22 +143,15 @@ describe('EarnConfirmationScreen', () => {
     await waitFor(() => {
       expect(queryByTestId('EarnConfirmation/GasLoading')).toBeFalsy()
     })
+
     expect(getByTestId('EarnConfirmation/GasFeeCryptoAmount')).toHaveTextContent('0.06 ETH')
     expect(getByTestId('EarnConfirmation/GasFeeFiatAmount')).toHaveTextContent('₱119.70')
     expect(queryByTestId('EarnConfirmation/GasSubsidized')).toBeFalsy()
     expect(getByTestId('EarnConfirmationScreen/CTA')).toBeEnabled()
-    expect(prepareWithdrawAndClaimTransactions).toHaveBeenCalledWith({
```

Comment:
Can you verify that the appropriate prepareTransaction function is called with the expected arguments?

Reply:
Added in 26316f89a7e4947617fa7a1a01659e44823bb167!